Templates/Other/LinuxMain.stencil

```diff
@@ -0,0 +1,14 @@
+import XCTest
+@testable import AbstractTests
+
+XCTMain([
+{% for type in types.based.XCTestCase %}
+	testCase({{ type.name }}.allTests),
```

Comment:
Do you mean in the Stencil templates? Why? I don't have strong feelings about this but I indent with tabs in all of the code I write, and the code in Abstract uses tabs. Is there a particular reason to use spaces instead of tabs in this context?